### DISCUSSION TITLE
{【台服】遇上了奇怪的bug(?

### DISCUSSION THREAD

**Discussion by ethan940726** [MaaAssistantArknights/MaaAssistantArknights#6361] [Q&A] [answered]
我是台服的玩家，平常電腦內就有3套的模擬器+MAA分別配對。台服更新敘拉古人以後突然有其中一組出現了問題，MAA啟動之後只會不斷點擊模擬器左下角之位置，不會執行程序。
我已嘗試過自行排解，adb確認連接沒有問題，硬體效能也完全足夠，也已經更新到最新版本。同時也看過"常見問題-連接正常，但是無操作"與"連接正常，但是操作卡頓、異常或頻繁出錯"並根據步驟重新設定過，但依然無法解決，想請問這是甚麼問題呢?
(出現異常前並沒有更改任何設定，之前也未出現過此問題。同時，另二模擬器(台服)依舊能正常運行)

![螢幕擷取畫面 2023-09-14 151316](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/71484388/66885b3f-721a-444a-8123-8224429766c0)
abd設定狀態

![螢幕擷取畫面 2023-09-14 151557](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/71484388/96399b09-bb9c-4b5a-a42c-75abefb7345a)
MAA與模擬器運行時狀況

如果能協助解決，在此表示萬分感謝!

**Comment by ABA2396** [comment, marked as answer]
到主仓库的issue里发吧，顺便带上debug文件夹里的两个日志。能圈一下点击的位置吗？感觉像是模拟器给传的图片颠倒了。

**Reply by ethan940726** [reply]
好的，謝謝!
